classes/Hook/HookDisplayBackOfficeHeader.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Ps_Googleanalytics\Hooks;
+
+use Configuration;
+use Context;
+use Order;
+use PrestaShop\Module\Ps_Googleanalytics\Handler\GanalyticsJsHandler;
+use PrestaShop\Module\Ps_Googleanalytics\Repository\GanalyticsRepository;
+use PrestaShop\Module\Ps_Googleanalytics\Wrapper\OrderWrapper;
+use Ps_Googleanalytics;
+use Tools;
+use Validate;
+
+class HookDisplayBackOfficeHeader implements HookInterface
+{
+    private $module;
+    private $context;
+
+    public function __construct(Ps_Googleanalytics $module, Context $context)
+    {
+        $this->module = $module;
+        $this->context = $context;
+    }
+
+    /**
+     * run
+     *
+     * @return string
+     */
+    public function run()
+    {
+        $js = '';
+        if (strcmp(Tools::getValue('configure'), $this->module->name) === 0) {
+            $this->context->controller->addCSS($this->module->getPathUri() . 'views/css/ganalytics.css');
+        }
+
+        $ga_account_id = Configuration::get('GA_ACCOUNT_ID');
+
+        if (!empty($ga_account_id) && $this->module->active) {
+            $gaTagHandler = new GanalyticsJsHandler($this->module, $this->context);
+            $this->context->controller->addJs($this->module->getPathUri() . 'views/js/GoogleAnalyticActionLib.js');
+
+            $this->context->smarty->assign('GA_ACCOUNT_ID', $ga_account_id);
+
+            $gaScripts = '';
+            if ($this->context->controller->controller_name == 'AdminOrders') {
```

Comment:
To be checked with 1.7.7
I think it works, we are keeping `$this->context->controller->controller_name` compatible ... but better double-check to be sure 😄

Reply:
I checked in 1.7.7 and I had no issue. What do you mean by `double-check to be sure` ?

Reply:
Double-check:
1st check: me ; I think it works
2nd check: you

So both of us have checked, it's 👍